libdrizzle/structs.h

```diff
@@ -168,7 +172,11 @@ struct drizzle_options_st
     interactive(false),
     multi_statements(false),
     auth_plugin(false),
-    socket_owner(DRIZZLE_SOCKET_OWNER_NATIVE)
+    socket_owner(DRIZZLE_SOCKET_OWNER_NATIVE),
+    wait_timeout(DRIZZLE_DEFAULT_SOCKET_TIMEOUT),
+    keepidle(5),
```

Comment:
There are default values set by the system. Which we could use. Might be better in terms of having a "standard" linux configuration  when the values are not set explicitly
```
tcp_keepalive_time = 7200 (seconds)
tcp_keepalive_intvl = 75 (seconds)
tcp_keepalive_probes = 9 (number of probes)
```